### PR TITLE
Add NodeJS v8.2.1 support

### PR DESCRIPTION
**Pull request by yellowred**
This adds NodeJS v8.2.1 support.

See https://github.com/strongloop/loopback-context/issues/32

### CODE REVIEW

**Comment by slnode** [comment]
Can one of the admins verify this patch? To accept patch and trigger a build add comment ".*ok\W+to\W+test.*"

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by slnode** [comment]
Can one of the admins verify this patch?

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+2017-07-21, Version 3.3.0
+=========================
+
+ * Add Node.js 8.2.1 support (Oleg Kubrakov)
```
Comment (bajtos) [inline CHANGES.md]: This file is autogenerated as part of our release process, please revert these changes.

**Review by bajtos — CHANGES_REQUESTED** [review]
Lovely!

The changes look good to me, with the exception of CHANGES.md (see my comment below). Could you please revert them?

When you are done, please squash all commits into a single one:

 - `git rebase -i master`
 - follow the help displayed in the editor, use `squash` or `fixup` for the second (and third?) commit
 - when `git` offers you the final commit message, just remove the extraneous content
 - once `git rebase` is done, you need to force-push the changes to override git history on the server:
 - `git push -f`

If things don't work out as expected, you can always `git reset --hard 4642a46` to get back to the state as I see in this pull request now.

If you happen to find this history manipulation too difficult, then I can make the changes for you, but I think this is a good learning opportunity and therefore I prefer you to at least try :)

Let me know when you are done, so that I can land and release this.

**Comment by yellowred** [comment]
done:
- changes reverted,
- commits squashed.

**Review by bajtos — APPROVED** [review]
LGTM 👍

**Comment by bajtos** [comment]
Landed and released in `loopback-context@3.3.0`, thank you for the contribution! 🎉
